integrations/src/test/java/io/jenkins/plugins/casc/SchemaTest.java

```diff
@@ -0,0 +1,72 @@
+package io.jenkins.plugins.casc;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.gargoylesoftware.htmlunit.WebRequest;
+import com.gargoylesoftware.htmlunit.WebResponse;
+import io.jenkins.plugins.casc.misc.ConfiguredWithCode;
+import io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule;
+import org.jenkinsci.plugins.pipeline.modeldefinition.shaded.com.fasterxml.jackson.databind.JsonNode;
+import org.jenkinsci.plugins.pipeline.modeldefinition.shaded.com.github.fge.jsonschema.main.JsonSchemaFactory;
+import org.jenkinsci.plugins.pipeline.modeldefinition.shaded.com.github.fge.jsonschema.main.JsonValidator;
+import org.jenkinsci.plugins.pipeline.modeldefinition.shaded.com.github.fge.jsonschema.report.ProcessingReport;
+import org.jenkinsci.plugins.pipeline.modeldefinition.shaded.com.github.fge.jsonschema.util.JsonLoader;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import static com.gargoylesoftware.htmlunit.HttpMethod.POST;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+public class SchemaTest {
+
+    @Rule
+    public JenkinsConfiguredWithCodeRule j = new JenkinsConfiguredWithCodeRule();
+
+    @Test
+    @ConfiguredWithCode("SchemaTest.yml")
+    public void schemaConfigurationTest() throws Exception {
+        JenkinsRule.WebClient client = j.createWebClient();
+        WebRequest request =
+                new WebRequest(client.createCrumbedUrl("configuration-as-code/schema"), POST);
+        WebResponse response = client.loadWebResponse(request);
+        assertThat(response.getStatusCode(), is(200));
+        String schema = response.getContentAsString();
+        String contents = new String(Files.readAllBytes(Paths.get("./src/test/resources/io/jenkins/plugins/casc/SchemaTest.yml")));
```

Comment:
you should be able to get this as a resource, 
something like:
```suggestion
        String contents = new String(Files.readAllBytes(getClass().getResource("SchemaTest.yml")));
```

Reply:
` String contents = new String(Files.readAllBytes(Paths.get(getClass().getResource("SchemaTest.yml").toURI())));`
Cool, somtheing like this works.

Reply:
Hmmm we already have a helper function inside the util package

Reply:
https://github.com/jenkinsci/configuration-as-code-plugin/blob/aca309053acae7fc8227c91c82c165f6e6e7f6a1/plugin/src/test/java/io/jenkins/plugins/casc/misc/Util.java#L155